Views/Lombiq.Privacy.ConsentBanner.cshtml

```diff
@@ -1,6 +1,8 @@
 @using Microsoft.AspNetCore.Http.Features;
 @using Lombiq.Privacy.Controllers;
 
+@inject Microsoft.AspNetCore.Antiforgery.IAntiforgery antiForgery
```

Comment:
The injected service should be PascalCase, because in the generated code it becomes a property.